content/blog/2021-02-04-skywalking-trace-threadpool/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Apache SkyWalking: How to propagate context between threads when using ThreadPoolExecutor"
+date: 2021-02-04
+author: "Binglong Li"
+description: "This post introduces how to propagate context between threads when using ThreadPoolExecutor, 
+which skywalking agent can not enhance"
+tags:
+- Java
+- Agent
+- Tracing
+- thread
```

Comment:
```suggestion
```

Reply:
What about adding a ThreadPoolExecutor tag?

Reply:
tag is only for widely used in multiple blogs.